src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,162 @@
+
+Orphan spans refer to spans that exist or are visible, but their parent spans are not visible or are not within Sentry's database. This could imply a bug in the Sentry application. If you're confident about this, please contact support: [support@sentry.io](mailto:support@sentry.io)
+
+Another reason could include that instrumented spans may not make it back to Sentry for processing. As an example, let us refer back to our earlier example of the web application consisting of:
+
+- Frontend (Single-Page Application)
+- Backend
+- Background Queue
+- Notification Job
+
+Instrumented spans in each component are sent back to Sentry for processing. However, a server in the backend component could suffer from network connectivity loss, and the spans may not have made it back to Sentry. This would indicate a "hole" within this specific trace. The collected spans within the Search Service would be descendants of a span (for example, a search service call) within the Backend component, and thus the top-level span of the collective Search Service spans is referred to as an orphan span.
+
+### Properties
+
+Each span has a unique identifier kept in the `span_id` attribute.
+
+Start time and end time are stored in the `start_timestamp` and `timestamp` attributes, respectively. The format is an RFC 3339 / ISO 8601-compatible string or a numeric value with the number of seconds since the UNIX Epoch.
+
+The name of the operation is stored in the “op” parameter. Examples are `http`, `sql.query`, `redis.command`.
+
+Additional data about the operation can be stored in the `data` and `tags` attributes.
+
+## Trace
+
+A trace is a collection of spans that have the same `trace_id` attribute. Traces consist of all spans generated by the distributed system during the processing of the input data. Traces are not explicitly started or finished. The start and end of a trace are defined by the transactions and spans participating in the trace.
+
+Traces that start with the same set of API endpoint calls can potentially differ in terms of what code paths were executed. For example, an initial trace may result in a cache miss at the Cache Service, and a subsequent trace may include more information instrumented at the Cache Service. Although these two traces are similar, they are not identical.
+
+### Propagation
+
+Processing of the input data in the distributed environment means that an arbitrary number of the systems/microservices can be involved.
+
+To connect all the spans that were created during the processing of the input data, we need to be able to propagate the basic identifier. By forwarding `trace_id` and `parent_span_id` as data travels through various applications in a microservice ecosystem, we can reconstruct data flows later.
+
+Trace metadata is propagated through http headers that are included in the http request to the external system.
+
+## Transaction
+
+There already was a concept of transaction in the Sentry SDKs before we started with APM.
```

Comment:
```suggestion
```

I think we can omit this line altogether.

Reply:
@HazAT @untitaker was this something tracked by the SDKs itself prior to tracing?